pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -162,14 +162,19 @@ func (mi *StorageInitializerInjector) InjectStorageInitializer(pod *v1.Pod) erro
 		storageInitializerImage = mi.config.Image
 	}
 
+	modelLocalMountPath := constants.DefaultModelLocalMountPath
+	if modelName, ok := pod.ObjectMeta.Labels[constants.InferenceServicePodLabelKey]; ok {
```

Comment:
`modelName` maybe not be always same as service name, custom container can pass `modelName` as an argument